### PR TITLE
add home page distance filter button and profile page styling

### PR DESCRIPTION
**Pull request by LukeBooker**
Added some styling and functionality to the profile page - a user can flip pet cards vertically. In the home page, added filter by distance and cleaner styling on the drop downs and search bar.

### CODE REVIEW

File: lib/pages/login_page.dart [hunk 1]
```diff
@@ -66,27 +71,36 @@ class _LoginPageState extends State<LoginPage> {
...
 
   Widget _logoWidget() {
-    return SizedBox(
-        height: _deviceHeight! * 0.5,
-        child: Image.asset(
-          'ptp_logocardsglow.png',
-          fit: BoxFit.cover,
-        ));
+    return Container(
+      margin: EdgeInsets.fromLTRB(0, 0, 0, 10),
```
Comment (IanFindlay) [inline lib/pages/login_page.dart:75]: const needed for the EdgeInsets

File: lib/pages/nav_home_page.dart [hunk 2]
```diff
@@ -42,13 +42,18 @@ class _NavHomePageState
...
+                  const Color.fromARGB(
+                      255, 83, 167, 245),
               centerTitle: true,
               title: Image.asset(
                   'ptp_logolong.png',
                   height: 40,
                   fit: BoxFit.cover),
             )
           : AppBar(
+              backgroundColor: Color.fromARGB(
```
Comment (IanFindlay) [inline lib/pages/nav_home_page.dart:55]: const

File: lib/pages/nav_home_page.dart [hunk 3]
```diff
@@ -68,7 +73,7 @@ class _NavHomePageState
         ],
         currentIndex: _selectedIndex,
         selectedItemColor:
-            Color.fromARGB(255, 255, 53, 53),
+            Color.fromARGB(255, 83, 167, 245),
```
Comment (IanFindlay) [inline lib/pages/nav_home_page.dart:76]: const

File: lib/pages/nav_home_page.dart [hunk 4]
```diff
@@ -12,7 +12,7 @@ class NavHomePage extends StatefulWidget {
 
 class _NavHomePageState
     extends State<NavHomePage> {
-  int _selectedIndex = 0;
+  int _selectedIndex = 1;
   String _appBarName = "Home";
 
   static const TextStyle optionStyle = TextStyle(
```
Comment (IanFindlay) [inline lib/pages/nav_home_page.dart:18]: This optionStyle doesn't seem to be used anywhere - can it be removed? I don't think it is something you've done but might as well get cleaned up now.

File: lib/pages/profile_page.dart [hunk 5]
```diff
@@ -9,31 +10,111 @@ class ProfilePage extends StatefulWidget {
   }
 }
 
-class _ProfilePageState extends State<ProfilePage> {
+class _ProfilePageState
+    extends State<ProfilePage> {
   double? _deviceHeight;
   double? _deviceWidth;
```
Comment (IanFindlay) [inline lib/pages/profile_page.dart:16]: Again, not something this PR did but _deviceWidth isn't used as far as I can tell as it is redefined in the build

File: lib/pages/profile_page.dart [hunk 6]
```diff
@@ -72,100 +155,209 @@ class _ProfilePageState extends State<ProfilePage> {
...
-            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-            mainAxisSize: MainAxisSize.max,
-            crossAxisAlignment: CrossAxisAlignment.center,
-            children: [
-              InkWell(
-                onTap: () {},
-                child: const Icon(
-                  Icons.delete,
-                  color: Colors.red,
+      padding: EdgeInsets.fromLTRB(5, 10, 5, 0),
```
Comment (IanFindlay) [inline lib/pages/profile_page.dart:209]: const

File: lib/pages/profile_page.dart [hunk 7]
```diff
@@ -190,20 +387,25 @@ class _ProfilePageState extends State<ProfilePage> {
 
   Widget _addButton() {
     return SizedBox.fromSize(
-      size: const Size(45, 45),
+      size: const Size(50, 50),
       child: ClipOval(
         child: Material(
-          color: Colors.purple,
+          color: Color.fromARGB(255, 236, 68, 68),
```
Comment (IanFindlay) [inline lib/pages/profile_page.dart:393]: const

File: lib/pages/profile_page.dart [hunk 8]
```diff
@@ -220,58 +422,67 @@ class _ProfilePageState extends State<ProfilePage> {
...
-            "Bobby11 - 12/03/2022 13:02",
-            style: TextStyle(
-              fontWeight: FontWeight.w600,
-            ),
-          ),
-          Text(
-              "Rover is the devil. He ate my piglet! Avoid at all cost. Jess is hot though"),
-        ],
-      ),
+      padding: EdgeInsets.fromLTRB(30, 0, 30, 0),
```
Comment (IanFindlay) [inline lib/pages/profile_page.dart:458]: const

File: lib/pages/profile_page.dart [hunk 9]
```diff
@@ -220,58 +422,67 @@ class _ProfilePageState extends State<ProfilePage> {
...
+            return Container(
+              margin: const EdgeInsets.only(
+                  bottom: 5, top: 5),
+              child: Column(
+                crossAxisAlignment:
+                    CrossAxisAlignment.start,
+                children: [
+                  Text(
+                    "From: ${entries[0]["reviews"][index]["fromUser"]} ${entries[0]["reviews"][index]["date"]}",
+                    style: TextStyle(
```
Comment (IanFindlay) [inline lib/pages/profile_page.dart:476]: const

**Review by IanFindlay — CHANGES_REQUESTED** [review]
Other than missing a few const this is a great PR. Couple of things to clean up in the file from previous PRs while we are here but this is basically ready to merge

**Comment by LukeBooker** [comment]
Thanks, Ian! All amended.

**Review by IanFindlay — APPROVED** [review]

**Pull request merged**